LostTreasure/README

```diff
@@ -1 +0,0 @@
-This Add-on is not created by, affiliated with or sponsored by ZeniMax Media Inc. or its affiliates. The Elder Scrolls® and related logos are registered trademarks or trademarks of ZeniMax Media Inc. in the United States and/or other countries. All rights reserved.
```

Comment:
This disclaimer has to be here iirc. Why is it removed

Reply:
it's in the manifest file already. just thought it's not required to write it twice.